R/correlations.R

```diff
@@ -21,8 +21,6 @@
 correlation <- function(scores,
                         metrics = NULL,
                         digits = NULL) {
-  metrics <- check_metrics(metrics)
```

Comment:
Is this unrelated to this PR?

Reply:
Yes, but I noted this! 🥺 🐶 It's mentioned in the PR description and you asked me to fix these 🐩

Reply:
Why is it not its own PR?